client/src/Routes.jsx

```diff
@@ -72,16 +72,16 @@ function Routes() {
   const { state, dispatch } = useContext(GlobalContext);
 
   useEffect(() => {
-    try {
-      if (state.user && Object.keys(state.user).length) {
+    if (state.user && Object.keys(state.user).length) {
```

Comment:
put this if statement in try catch, if it throws error you wont be able to handle it